src/BBT.StructureTools/Initialization/IIocResolver.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright © BBT Software AG. All rights reserved.
+
+namespace BBT.StructureTools.Initialization
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// An interface that has to implement by user of this project.
```

Comment:
Describe what the interface does not who needs to implement it (this should be part of the documentation)